src/screens/NewAnalytics/PaymentAnalytics/PaymentsSuccessRate/PaymentsSuccessRateUtils.res

```diff
@@ -0,0 +1,17 @@
+open NewPaymentAnalyticsUtils
+
+let getPaymentQueryDataString = queryData =>
+  switch queryData {
+  | #PaymentSuccessRate => "payments_success_rate"
```

Comment:
`payments_success_rate` is aloready defined as a metric type in NewAnalyticsTypes

Reply:
let us use it from the respective module utils file itself